actors/common/src/main/java/org/sunbird/metrics/actors/OrganisationMetricsActor.java

```diff
@@ -71,7 +72,9 @@ public void onReceive(Request request) throws Throwable {
   }
 
   private void orgConsumptionMetricsReport(Request actorMessage) {
-    ProjectLogger.log("OrganisationMetricsActor-orgConsumptionMetricsReport called");
+    ProjectLogger.log(
+        "OrganisationMetricsActor:orgConsumptionMetricsReport method call start.",
```

Comment:
From:
"OrganisationMetricsActor:orgConsumptionMetricsReport method call start."

To:
"OrganisationMetricsActor: orgConsumptionMetricsReport called."